lnd/lnd.go

```diff
@@ -205,12 +204,16 @@ func (lnd *LND) ListChannels() ([]*lightning.LightningChannel, error) {
 }
 
 func (lnd *LND) CreateInvoice(value int64, preimage []byte, expiry int64, memo string) (*lightning.AddInvoiceResponse, error) {
-	invoice, err := lnd.client.AddInvoice(lnd.ctx, &lnrpc.Invoice{
+	request := &lnrpc.Invoice{
 		Memo:      memo,
 		Value:     value,
 		Expiry:    expiry,
 		RPreimage: preimage,
-	})
+	}
+	if lnd.regtest {
+		request.CltvExpiry = 20
```

Comment:
This 20 magic value is used for LND and CLN. Let's make it a constant used by both
And add a comment why this is being done on regtest only